### PR TITLE
NXP-16827 Make default SI binary prefix configurable and its usage co…

### PR DESCRIPTION
**Pull request by ron1**
Make default SI binary prefix configurable and its usage consistent

### CODE REVIEW

File: nuxeo-distribution/nuxeo-distribution-resources/src/main/resources/templates-common/common/config/nuxeo.properties [hunk 1]
```diff
@@ -30,3 +30,6 @@ org.nuxeo.ecm.platform.liveedit.config=client
 
 # size limit to use the specific download servlet for big files (use SI or IEC prefix)
 #org.nuxeo.big.file.size.limit=5Mi
+
+# default byte prefix format (SI/JEDEC/IEC)
+#org.nuxeo.defaultBytePrefixFormat
```
Comment (jcarsique) [inline nuxeo-distribution/nuxeo-distribution-resources/src/main/resources/templates-common/common/config/nuxeo.properties]: `#org.nuxeo.defaultBytePrefixFormat=SI` (to highlight the default value)

Comment (ron1) [inline nuxeo-distribution/nuxeo-distribution-resources/src/main/resources/templates-common/common/config/nuxeo.properties]: Yes, my bad for omitting the '=SI'. Would you like me to submit another pull request with this modification?

**Comment by atchertchian** [comment]
Hi,
It's been merged with some adaptations for https://jira.nuxeo.com/browse/NXP-17673, see https://jira.nuxeo.com/browse/NXP-16827.
Thanks for the pull request!
